Drivers/c/LidarExample/main.c

```diff
@@ -0,0 +1,175 @@
+#include "FreeRTOS.h"
+#include "task.h"
+
+#include <Hal/Hal.h>
+#include <Init/Init.h>
+
+#include <RegisterHwas/RegisterHwas.h>
+#include <UartHwas/UartHwas.h>
+#include <UartHwas/UartHwasRegisters.h>
+
+#include <stdio.h>
+#include <string.h>
+
+#define LIDAR_RX_BUFFER_LEN 100
+uint8_t lidarRxBuffer[LIDAR_RX_BUFFER_LEN] = {""};
+
+ByteFifo rxFifo;
+UartHwas uart;
+
+#define PRV_INTERRUPT_STACK_SIZE 400
+__attribute__((section(".sdramMemorySection"))) static StackType_t
+    prvTaskStackBuffer[PRV_INTERRUPT_STACK_SIZE];
+__attribute__((
+    section(".sdramMemorySection"))) static StaticTask_t prvTaskBuffer;
+
+#define TF_LUNA_DATA_FRAME_HEADER1 0x59
+#define TF_LUNA_DATA_FRAME_HEADER2 0x59
+#define TF_LUNA_DATA_FRAME_LENGTHT 9
+
+size_t txIndex = 0;
+size_t txLength = 0;
+uint8_t txBuffer[20];
+
+static void prvTask(void *pvParameters);
+
+void lidarRxCallback(uint8_t readByte) {
+  if (!ByteFifo_push(&rxFifo, readByte)) {
+    Hal_console_usart_write("Fifo Full\n", 10);
+  }
+}
+
+void lidarTxCallback(void) {
```

Comment:
Is it used?

Reply:
No, I will delete that.

Reply:
Still unused, but OK, let's prioritize other issues.